app/src/main/res/values/strings.xml

```diff
@@ -44,4 +44,5 @@
     <string name="player_avatar">Player avatar</string>
     <string name="sign_in">Sign in</string>
     <string name="pgs">PGS</string>
```

Comment:
this may be a bit later, but we shouldn't use PGS. We should have Play Games Services as many developers are not really aware of PGS

Reply:
True, we don't usually use "PGS" as an official acronym, please use "Play Games Services" when referring to our product publicly.